.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+dist: bionic
+before_install:
+  - sudo apt-get update
+  - sudo apt-get -y install gnupg2 libgpgme-dev swig
+install:
+  - pip install --upgrade poetry
+  - poetry install
+matrix:
+  include:
+  - name: "Lint and static analysis"
+    python: 3.7
+    script:
+      - poetry run ./scripts/copyright_line_check.sh
+      - poetry run black --check --config pyproject.toml .
+      - poetry run flake8 --config setup.cfg . d
+      - poetry run isort --check --settings-path setup.cfg -rc .
+      - poetry run pydocstyle config=setup.cfg .
+      - poetry run pylint --rcfile=.pylintrc .
+  - name: "Unit Tests"
+    python: 3.7
+    script:
+      - poetry run python -m pytest .
```

Comment:
Perhaps in a separate PR, I'd add tests with all python versions that this package supports. Presumably that's 3.8 as well, and possibly 3.6 too?